op_builder/unittest_builder.py

```diff
@@ -0,0 +1,15 @@
+from .transformer_inference import InferenceBuilder
+
+
+class UnitTestBuilder(InferenceBuilder):
+    NAME = "deepspeed_unittest"
+
+    def __init__(self, name=None):
```

Comment:
The `__init__` implementation looks like matches the InferenceBuilder exactly, do we need to override it at all here?

Reply:
removed